pages/index.html

```diff
@@ -17,112 +17,87 @@
   or <a href="http://homotopytypetheory.org/coq/">homotopy type theory</a>),
   and <a href="/cocorico/CoqInTheClassroom">teaching</a>.
   </p>
+
+  <p>The reference documentation for Coq are the <a
+  href="/distrib/current/refman/"> Reference Manual </a> and the
+  documentation of the <a href="/distrib/current/stdlib/">
+  Standard Library</a>.
+  However, to learn about Coq, it's best to start with a tutorial
+  or book. Go to our <a href="/documentation">documentation page</a>
+  for a list of such tutorials and books.</p>
+
 </div>
 
 <div class="frameworklinks">
 <ul>
-<li><a href="/about-coq"> More about Coq </a></li>
+<li><a href="/documentation">Tutorials and books</a></li>
+<li><a href="/distrib/current/refman/"> Reference Manual </a></li>
+<li><a href="/distrib/current/stdlib/"> Standard Library </a></li>
 </ul>
 </div>
 </div>
 
-
 <div class="framework">
-<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/download">How to get it?</a></div>
+<div class="frameworklabel"><a style="color:black;font-weight:bold" href="/download">Try / Install Coq</a></div>
 <div class="frameworkcontent">
-  <p>You can download <a href="<#CURRENTVERSIONURL>">the current stable version,
-      Coq <#CURRENTVERSION></a>, released in <#CURRENTVERSIONDATE>.</p>
-<#ifdef UPCOMINGVERSION>
-  <p><a href="<#UPCOMINGVERSIONURL>">The upcoming version, Coq <#UPCOMINGVERSION></a>
-    is also available for testing.</p>
-</#ifdef>
-<p>
-To use Coq, you will need a user interface.  Many editor support
-extensions are available (for Emacs, Vim, VSCode, etc.), as well as
-CoqIDE, a standalone IDE for Coq.
-The <a href="user-interfaces.html">User interfaces</a> page provides a
-full list, with more details.  If you are only looking for a quick way
-to try Coq without installing anything, we recommend you
-use <a href="https://jscoq.github.io">jsCoq</a>.
-</p>
+<p>You don't need to install Coq to get started: run Coq in your browser using
+<a href="https://jscoq.github.io">jsCoq</a>!</p>
+
+<p>To go beyond, you can proceed to <a href="/download">install the
+Coq platform</a> on your machine and
+a <a href="user-interfaces.html">user interface</a> of your
+choice.</p>
```

Comment:
>I expect most people would eventually install Coq on their own system. Is that right? Can we give a sentence saying why users will want to install on their system--what added features, etc.?

That's right. Getting Coq installed gives you a proper way to conduct projects with multiple files, gives you access to faster reduction machines, etc.

>BTW, I find it very hard to pick out the links in the text. Can you change the highlight color for all the links to something more vivid?

Again a design issue. I'd rather not touch that in this PR, especially as @maximedenes is planning to prepare a refresh of the design website.

Reply:
If you're willing to ask Maxime and he's O with it, you could increase the red level.  Should be one line in a CSS file.  It would affect all pages in the website.  The links are barely discernable--it's an easy way to make it a lot more usable.

If I may offer an opinion, it would nice if the developers were invited to offer suggestions on the website (but probably leaving the decisions to Maxime).  Also incremental improvement and review is a superior to the "waterfall" development model (both for websites and software).